onnx/onnxifi_dummy.c

```diff
@@ -0,0 +1,112 @@
+/*
+ * Dummy implementation of ONNX backend interface for manual test.
+ * Prints the name of the called function and backend name on each call.
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+
+#include "onnxifi.h"
+
+ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI
+onnxGetBackendIDs(onnxBackendID* backendIDs, size_t* numBackends) {
+  *numBackends = 2;
+  *backendIDs = 0;
```

Comment:
could you change numBackends to 1? right now you are setting only one backendID

Reply:
ok, hhh